http/handler.go

```diff
@@ -1610,19 +1610,16 @@ func (h *Handler) handlePostImportRoaring(w http.ResponseWriter, r *http.Request
 			w.WriteHeader(http.StatusInternalServerError)
 		}
 	}
+
 	// Marshal response object.
```

Comment:
i don't immediately see why these spans are going away. (also the blank line may be unintentional?)

Reply:
the response to an import request is tiny, so I didn't think it was worth the overhead and line noise to have separate spans around serializing and writing back. The entirety of the request is still within a span, so it will be fairly obvious if these last bits are taking up a lot of time for some reason.